api-reference/10 UI Widgets/dxScheduler/1 Configuration/recurrenceEditMode.md

```diff
@@ -9,16 +9,30 @@ default: 'dialog'
 Specifies the edit mode for recurring appointments.
 
 ---
-This option accepts the following values.
+This option accepts the following values:
 
-- "dialog"  
- Displays a dialog that suggests to a user to choose between editing the entire series or only the current appointment.
+- *"series"*  
+ Enables a user to edit only the entire appointment series.
 
-- "series"  
- Enables an end-user to edit only the entire appointment series.
+- *"occurrence"*  
+ Enables a user to edit only the individual appointment instance.
 
-- "occurrence"  
- Enables an end-user to edit only the current appointment.
+- *"dialog"*
+ Displays a dialog that suggests to a user to choose between editing the entire series or only the individual instance.
+
+Changes made to recurring appointment instance and series are handled differently.
```

Comment:
```suggestion
The **Scheduler** handles changes made to an instance and changes made to a series differently.
```

Reply:
The **Scheduler** handles changes made to an instance and a series differently. Is it better?

Reply:
Ok